arviz/plots/backends/matplotlib/distplot.py

```diff
@@ -86,20 +93,50 @@ def plot_dist(
     return ax
 
 
-def _histplot_mpl_op(values, values2, rotated, ax, hist_kwargs):
+def _histplot_mpl_op(values, values2, rotated, ax, hist_kwargs, is_circular):
     """Add a histogram for the data to the axes."""
+
+    bins = hist_kwargs.pop("bins")
+
+    if is_circular:
+
+        if values.min() < np.pi and values.max() > np.pi:
+            if bins is None:
+                bins = get_bins(values)
+            values = np.deg2rad(values)
+            bins = np.deg2rad(bins)
+        else:
+            label = [
+                r"$0$",
```

Comment:
This requires the user to have LaTeX installed, right? Should we use unicode instead?

Reply:
No, it seems Latex is not needed.